### ISSUE TITLE
ThreeSD won't detect content in the SD card

### ISSUE THREAD

**Issue by Seamed** [zhaowenlan1779/threeSD#25]
I select the SD card, click ok, but the program says there're no contents availabl
![threesd_1](https://user-images.githubusercontent.com/103855819/163936433-c045c929-c034-4db6-8e94-01d29e6793b0.PNG)
![threesd_2](https://user-images.githubusercontent.com/103855819/163936445-af36e118-e904-4ad1-8d65-7d0961d6aaf9.PNG)
e

**Comment by zhaowenlan1779** [comment]
Hello, sorry that I didn't see this for such a long time. Could you please post a log? It should be in the same folder as the threeSD executable.

**Comment by Seamed** [comment]
Hey, sorry for leaving you hanging myself here's the log
[threeSD_log.txt](https://github.com/zhaowenlan1779/threeSD/files/8748588/threeSD_log.txt)

**Comment by CROTEX** [comment]
Same issue as above here

**Comment by TromboneFreak** [comment]
Same @zhaowenlan1779
[threeSD_log.txt](https://github.com/zhaowenlan1779/threeSD/files/9126707/threeSD_log.txt)

**Comment by DemonKingOdio** [comment]
Same, even tried downgrading to 2.0.0 as the script mentions
[threeSD_log.txt](https://github.com/zhaowenlan1779/threeSD/files/9367702/threeSD_log.txt)

**Comment by DemonKingOdio** [comment]
Could this possibly be from having too many savegames on the 3ds? I know I have 50 pages of 3ds titles alone

**Comment by VorgonBattlecruiser** [comment]
I have a lot of folders, too, because i downloaded a complete cheats archive, so this sounds like a possible reason.
Still cant use threeSD btw.

**Comment by zhaowenlan1779** [comment]
Thank you for reporting this issue. I'm sorry that I haven't been very active recently, but I'll try looking into it when I can.

**Comment by Xorathmion** [comment]
Yeah, I have the same issue. Followed the whole guide step by step, and I get the same issue on the ThreeSD.exe program when searching the Root of my SD card.
